src/components/Checkout/BasketItemsList.vue

```diff
@@ -20,7 +20,7 @@
 			<li v-for="donation in donations" :key="donation.id">
 				<donation-item
 					:donation="donation"
-					:loan-count="loans.length"
+					:loan-in-basket-count="loans.length"
```

Comment:
I don't this change was really necessary...but maybe update to "loans-in-basket-count"...

Reply:
Agreed, we can call it loanCount